homeassistant/components/cybro/translations/de.json

```diff
@@ -0,0 +1,26 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Ger\u00e4t ist bereits konfiguriert",
+            "cannot_connect": "Verbindung fehlgeschlagen"
+        },
+        "error": {
+            "cannot_connect": "Verbindung fehlgeschlagen"
+        },
+        "flow_title": "{name}",
+        "step": {
+            "user": {
+                "data": {
+                    "host": "Host",
+                    "port": "Port",
+                    "address": "Cybro 2/3 SPS NAD"
+                },
+                "description": "Richte deine Cybro SPS f\u00fcr die Integration mit Home Assistant ein."
+            },
+            "zeroconf_confirm": {
+                "description": "M\u00f6chtest du die Cybro SPS mit dem Namen `{name}` zu Home Assistant hinzuf\u00fcgen?",
+                "title": "Cybro SPS entdeckt"
+            }
+        }
+    }
+}
```

Comment:
Translations are not needed, they are added by 'Localize'. See https://developers.home-assistant.io/docs/translations/

You can remove the `translations` folder.

Reply:
Ok, i will do that

Reply:
If i delete the `translations` folder, and run `python3 -m script.translations develop` then the english translations are back again.
so for the moment they are still included.

Reply:
That's normal, you should only add the English version, the rest is done via localize.